examples/taint_checking/taint_checking.out

```diff
@@ -0,0 +1,8 @@
+# At the command line, navigate to the directory that contains
+# the `.bal` file and run the `ballerina run` command.
+$ ballerina run taint-checking.bal
```

Comment:
This should be "taint_checking.bal", please fix below other places as well.

Reply:
:hushed: Yes. Fixed in all locations.